docs/Spell_system.md

```diff
@@ -1,30 +0,0 @@
-##### This article was created and ported from Trinity core's wiki since their system is identical
```

Comment:
So no cofusion occurs, this is just a duplicate, and outdated, file of https://github.com/azerothcore/wiki/blob/master/docs/spell_system.md